packages/block-library/src/navigation/edit/index.js

```diff
@@ -136,13 +136,19 @@ function Navigation( {
 		`navigationMenu/${ navigationMenuId }`
 	);
 
-	const { innerBlocks, isInnerBlockSelected } = useSelect(
+	const { innerBlocks, isInnerBlockSelected, hasSubmenus } = useSelect(
 		( select ) => {
 			const { getBlocks, hasSelectedInnerBlock } = select(
 				blockEditorStore
 			);
+			const blocks = getBlocks( clientId );
+			const didFindSubmenu = !! blocks.find(
```

Comment:
Something about the variable "didFindSubmenu" feels curious to me. Could it just be called "submenus"?

Reply:
`hasSubmenus`

Reply:
I didn't use `hasSubmenus` because that's how I named the variable in the outer scope; I'll go with submenus.